mindsdb/libs/controllers/mindsdb_controller.py

```diff
@@ -24,13 +24,14 @@
 
 class MindsDBController:
 
-    def __init__(self, file=SQLITE_FILE):
+    def __init__(self, log_level=1, log_url='http://localhost:35261', file=SQLITE_FILE):
         """
-
         :param file:
         """
 
         self.setConfigs()
+        controller_uuid = str(uuid.uuid1())
+        log.initialize(controller_uuid, level=2, log_level, log_url)
```

Comment:
should not pass level=2 before positional arguments log_level, log_url

Reply:
Ups... apparently I had forgotten to push some changes and this stayed in there , my bad :s